test/src/main/java/io/strimzi/test/extensions/StrimziExtension.java

```diff
@@ -96,6 +96,7 @@
     /** Tags */
     public static final String ACCEPTANCE = "acceptance";
     public static final String REGRESSION = "regression";
+    public static final String FLAKY = "flaky";
```

Comment:
We should document each tag to say what the intended semantics are

Reply:
Done